q2_diversity/_alpha/_method.py

```diff
@@ -32,7 +32,7 @@ def non_phylogenetic_metrics():
             'dominance', 'doubles', 'enspie', 'esty_ci', 'fisher_alpha',
             'goods_coverage', 'heip_e', 'kempton_taylor_q', 'margalef',
             'mcintosh_d', 'mcintosh_e', 'menhinick', 'michaelis_menten_fit',
-            'observed_otus', 'osd', 'pielou_e', 'robbins', 'shannon',
+            'observed_features', 'osd', 'pielou_e', 'robbins', 'shannon',
```

Comment:
Please revert this change:

```suggestion
            'observed_otus', 'osd', 'pielou_e', 'robbins', 'shannon',
```